core/src/main/java/org/lflang/TargetConfig.java

```diff
@@ -25,24 +25,46 @@
 package org.lflang;
 
```

Comment:
Should we move this class and also `TargetProperty` and `TargetPropertyConfig` to the `target` package?

Reply:
Yes, good idea.